content/people/yoshitom.md

```diff
@@ -12,17 +11,19 @@ interests:
   - Representation learning
 education:
   - degree: Master of Applied Informatics
-    institution: University of California, Irvine.
+    institution: University of California, Irvine
     year: 2016
   - degree: B.Eng. in Computer and Information Science
-    institution: National Institute of Technology, Akashi College.
+    institution: National Institute of Technology, Akashi College
     year: 2014
 email: yoshitom@uci.edu
-website: http://labusers.net/~ymatsubara
+websites:
+  -title: Personal webpage
+   url: http://labusers.net/~ymatsubara
```

Comment:
An extra space of indentation is required here (before `url:`) and in
the line above (in between of `-` and `title:`).